Client/Ecosia/Experiments/Unleash/BingExperiment.swift

```diff
@@ -31,3 +31,20 @@ struct BingSearchExperiment {
         Analytics.shared.userSearchViaBingABTest()
     }
 }
+
+extension BingSearchExperiment {
```

Comment:
Any reason to create an extension for this and not add it to struct itself already in the file?

Reply:
Good question! I saw that as something added to the struct's default capabilities rather than something that struct would behave as default for